custom_components/adjustable_bed/beds/limoss.py

```
@@ -0,0 +1,606 @@
+        self._notify_callback: Callable[[str, float], None] | None = None
+        self._write_with_response: bool = True
+        self._counter: int = 0
+        self._capabilities_received: asyncio.Event | None = None
+        self._memory_slot_count: int = 0
+        self._reported_motor_count: int | None = None
+        self._last_positions_raw: dict[str, int] = {}
+        # Position normalization state (raw 32-bit samples -> estimated angles).
+        self._max_raw_estimate: dict[str, int] = dict(_DEFAULT_MAX_RAW_ESTIMATE)
+        self._low_sample_streak: dict[str, int] = {
+            motor: 0 for motor in _DEFAULT_MAX_RAW_ESTIMATE
+        }
```

Ruff (0.14.14)

[convention] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)